src/equations/3d/compressible_euler.jl

```diff
@@ -169,6 +169,42 @@ function initial_conditions_taylor_green_vortex(x, t, equation::CompressibleEule
   return prim2cons(SVector(rho, v1, v2, v3, p), equation)
 end
 
+function initial_conditions_blob(x, t, equation::CompressibleEulerEquations3D)
+  # blob test case, see Agertz et al. https://arxiv.org/pdf/astro-ph/0610051.pdf
+  # other reference: https://arxiv.org/pdf/astro-ph/0610051.pdf
+  # change discontinuity to tanh
+  # typical domain is rectangular, we change it to a square, as Trixi can only do squares
+  # resolution 128^3, 256^3
+  # domain size is [-20.0,20.0]^3
+  # gamma = 5/3 for this test case
+  R = 1.0 # radius of the blob
+  # background density
+  dens0 = 1.0
+  Chi = 10.0 # density contrast
+  # reference time of characteristic growth of KH instability equal to 1.0
+  tau_kh = 1.0
+  tau_cr = tau_kh/1.6 # crushing time
+  # determine background velocity
+  velx0 = 2*R*sqrt(Chi)/tau_cr
+  vely0 = 0.0
+  velz0 = 0.0
+  Ma0 = 2.7 # background flow Mach number Ma=v/c
+  c = velx0/Ma0 # sound speed
+  # use perfect gas assumption to compute background pressure via the sound speed c^2 = gamma * pressure/density
+  p0 = c*c*dens0/equation.gamma
+  # initial center of the blob
+  inicenter = [-15, 0, 0]
+  x_rel = x-inicenter
+  r = sqrt(x_rel[1]^2 + x_rel[2]^2 + x_rel[3]^2)
+  # steepness of the tanh transition zone
+  slope = 2
+  # density blob
+  dens = dens0 + (Chi-1) * 0.5*(1+(tanh(slope*(r+R)) - (tanh(slope*(r-R)) + 1)))
+  # velocity blob is zero
+  velx = velx0 - velx0 * 0.5*(1+(tanh(slope*(r+R)) - (tanh(slope*(r-R)) + 1)))
+  return prim2cons(SVector(dens, velx, vely0, velz0, p0), equation)
```

Comment:
Can we use `rho, v1, v2, v3, p` as the names of the primitive variables instead of `dens, velx, vely, velz, p`? That would be in accordance with the rest of the code - and good for Mr. Consistency :sunglasses:

Reply:
ok, changed.